src/helm/benchmark/static/schema.yaml

```diff
@@ -1662,6 +1684,14 @@ metric_groups:
       - name: classification_micro_f1
         split: ${main_split}
 
+  - name: multiple_choice_classification_metrics
```

Comment:
If the semantics of `multiple_choice_classification_macro_f1` are exactly the same as `classification_macro_f1`, and the semantics of `multiple_choice_classification_micro_f1` are exactly the same as `classification_micro_f1`, then I think you should delete this section and instead change `MultipleChoiceClassificationMetric` to output `classification_macro_f1` and `classification_micro_f1` instead [here](https://github.com/stanford-crfm/helm/blob/fce37d9dcb5fe9d863dce1ad0200b37c36c0f4ba/src/helm/benchmark/metrics/classification_metrics.py#L101-L106).

Sorry I missed this during the previous PR review.

Reply:
Thanks for pointing out this! We have updated `MultipleChoiceClassificationMetric` and deleted this section accordingly.